src/store/ViewStore.js

```diff
@@ -11,6 +11,7 @@ export default class ViewStore {
         when(() => this.context, this.onContextReady);
     }
 
+    staticViews = observable.box(ViewStore).get();
```

Comment:
very interesting design choice... could you explain your thought process on this?